src/FormField/RegisteredMFAMethodListField.php

```diff
@@ -70,11 +70,15 @@ public function getSchemaDataDefaults()
     /**
      * Get the registered backup method (if any) from the currently logged in user.
      *
+     * @param Member|null $member
      * @return RegisteredMethod|null
      */
-    protected function getBackupMethod($member = null): ?RegisteredMethod
+    protected function getBackupMethod(Member $member = null): ?RegisteredMethod
```

Comment:
Needs to be explicitly nullable not to cause issues with PHP 8.4
```suggestion
    protected function getBackupMethod(?Member $member = null): ?RegisteredMethod
```

Reply:
@michalkleiner I've since changed to requiring a Member object